packages/UI/src/components/Switch/Switch.stories.tsx

```diff
@@ -0,0 +1,45 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import Switch from './Switch';
+
+const meta: Meta<typeof Switch> = {
+  component: Switch,
+};
+
+export default meta;
+type Story = StoryObj<typeof Switch>;
+
+/*
+ *👇 Render functions are a framework specific feature to allow you control on how the component renders.
```

Comment:
Add Documentation here about Using an Label and htmlFor w/ id to link the two. 
One thing I noticed about both radix/and Shadcn/ui is that when you click the label, it also triggers a switch.

Reply:
good point, I'm having a [discussion with Anouar](https://www.figma.com/file/rvdnSOeHdjODE0U7xdNTR9?node-id=12:3800&mode=dev#697460378) on whether we should add a label component or not in Figma, great minds think alike!

htmlFor w/ id is something that should be known by developers generally since it's not custom made by us but it's still a good idea to add it once we have a `WithLabel` story.